source/reference/minio-mc/mc-batch-generate.rst

```diff
@@ -245,3 +256,93 @@ Sample YAML
 
 .. literalinclude:: /includes/code/replicate.yaml
    :language: yaml
+
+``keyrotate``
```

Comment:
Probably for a different PR but we should look into restructuring this page so we aren't digging down so far as to hit H4s

Reply:
I do agree. 
Added #827 to track.